saleor/csv/utils/export.py

```diff
@@ -57,6 +57,8 @@ def export_products(
     temporary_file.close()
 
     send_export_download_link_notification(export_file, "products")
+    manager = get_plugins_manager()
+    manager.gift_card_export_completed(export_file)
```

Comment:
A gift card event is called instead of a product event. Shouldn't the tests catch it?

Reply:
yes it should - but I see I also mocked a wrong method